Libraries/CMSIS/Device/Maxim/MAX32672/Include/sys_aes_regs.h

```diff
@@ -1,232 +0,0 @@
-/**
- * @file    sys_aes_regs.h
- * @brief   Registers, Bit Masks and Bit Positions for the SYS_AES Peripheral Module.
- */
-
-/* ****************************************************************************
```

Comment:
Just double checking here, this isn't entirely deleted, right? It's just renamed to aes_regs.h?

Reply:
Yes, the new UG will have AES instead of SYS_AES. See comment above for more info.